### PR TITLE
fix(action): removed unnecessary setOutput call

### PR DESCRIPTION
**Pull request by cmitzel-ncino**
I was hitting an issue with the action reporting:
```
Error: Unable to process file command 'output' successfully.
Error: Invalid format 'You have commit messages with errors
```
I looked into it and the setOutput() call was formatted wrong. I actually read through the action code again and realized the `generateOutputs.js` file was already setting that `results` output so my call to the core function was unnecessary. I removed it and updated the tests accordingly.

### CODE REVIEW

**Comment by cmitzel-ncino** [comment]
@wagoid this should fix the bug, but I'm going to test with my fork just to make certain.

File: src/action.js [hunk 1]
```diff
@@ -150,7 +150,7 @@ const showLintResults = async ([from, to]) => {
     // this would be a good place to implement the setNeutral() when it's eventually implimented.
     // for now it can pass with a check mark.
     console.log('Passing despite errors ✅')
-    setOutput(`You have commit messages with errors\n\n${formattedResults}`)
+    console.log(formattedResults)
```
Comment (wagoid) [inline src/action.js:153]: What about putting something like this instead? That way the last seen we see is the success emoji
```
console.log(`You have commit messages with errors\n\n${formattedResults}`)
console.log('Passing despite errors ✅')
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,73 +4,66 @@ All notable changes to this project will be documented in this file. See [commit
 
 ## [5.4.0](https://github.com/wagoid/commitlint-github-action/compare/v5.3.1...v5.4.0) (2023-04-10)
 
-
```
Comment (wagoid) [inline CHANGELOG.md:7]: Let's rollback these changes, as this file is automatically generated 💪

File: action.yml [hunk 3]
```diff
@@ -43,7 +44,7 @@ outputs:
     description: The error and warning messages for each one of the analyzed commits
 runs:
   using: docker
-  image: docker://wagoid/commitlint-github-action:5.4.0
+  image: Dockerfile
```
Comment (wagoid) [inline action.yml:47]: Did you add `Dockerfile` to test in your fork? The `commitlint` workflow does this too, it replaces `image` field with `Dockerfile` as part of the steps: https://github.com/wagoid/commitlint-github-action/blob/master/.github/workflows/commitlint.yml#L11

**Review by wagoid — COMMENTED** [review]
Nice catch! I forgot about what `setOutput` did and misunderstood your intentions, sorry for not catching during code review 😅

File: CHANGELOG.md (hunk 2 above)
Comment (cmitzel-ncino) [reply CHANGELOG.md:7]: 👍

Comment (cmitzel-ncino) [reply CHANGELOG.md:7]: do i need to do anything for this in this PR?

Comment (wagoid) [reply CHANGELOG.md:7]: Nop, it gets automatically generated from your commit messages 👍

**Pull request merged**